build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id "java"
-    id "org.springframework.boot" version "2.1.4.RELEASE"
+    id "org.springframework.boot" version "2.0.2.RELEASE"
```

Comment:
Reverting due to changed Spring Kafka startup behaviour, which means that until `KafkaContainer` is added to tests, the service won't start up.